std/format.d

```diff
@@ -157,6 +157,11 @@ $(I Width):
     $(I empty)
     $(I Integer)
     $(B '*')
+$(I Underscore):
+    $(I empty)
+    $(B '_')
+    $(B '_') $(I Integer)
+    $(B '_*')
```

Comment:
Also, ddoc is being stupid and rendering the `_` as blanks.  You have to write it as `__` (two underscores) instead.